### PR TITLE
replace social proof icon and removed its shadow

### PR DESCRIPTION
**Pull request by mohit-codecombat**
social proof icon is replaced with new images which are captured from figma design.

Live-coding-classes-from-CodeCombat-1002

Live-coding-classes-from-CodeCombat-1001

### CODE REVIEW

File: app/views/landing-pages/parents/PageParents.vue [hunk 1]
```diff
@@ -34,30 +34,24 @@
...
             </div>
 
             <div class="flex-spacer">
               <div class="codie-logo-container">
-                <img srcset="/images/pages/parents/2017_codie_award@1x.png 1x,
-                                        /images/pages/parents/2017_codie_award@2x.png 2x,
-                                        /images/pages/parents/2017_codie_award@3x.png 3x"
-                    src="/images/pages/parents/2017_codie_award@1x.png"
+                <img
+                    src="/images/pages/parents/2017_codie_award.png"
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:54]: Removing the img optimization without updating the static assets causes these to break:
image

I'm a little confused why tis was done. I believe the html changes can be reverted, as we only need the CSS changed.

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
I got a bit confused by the html image `src` changes. After reverting the html I think there may be a commit missing.
This PR doesn't include the new static images you sourced from Figma.
Once new images are included and old images are deleted I think this PR will be good to go!

Thank you!

File: app/views/landing-pages/parents/PageParents.vue (hunk 1 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue:54]: Images was missing so I committed new images and removed old images

**Review by AndrewJakubowicz — APPROVED** [review]
Looks excellent to me :D

**Pull request merged**